R/arg_reconcile.R

```diff
@@ -0,0 +1,117 @@
+#' @title Reconcile an argument list to any function signature.
+#' 
+#' @description Adapt an argument list to a function excluding arguments that 
+#'              will not be recognized by it, redundant arguments, and un-named
+#'              arguments. 
+#'
+#' @param fn      A function to which an argument list needs to be adapted. Use
+#'                the unquoted name of the function. If it's in a different 
+#'                package then the fully qualified unquoted name (e.g. 
+#'                \code{utils::read.table})
+#' @param ...     An arbitrary list of named arguments (unnamed ones will be 
+#'                ignored). Arguments in \code{.args} are overridden by
+#'                arguments of the same name (if any) in \code{...}
+#' @param .args   A list or \code{alist} of named arguments, to be merged 
```

Comment:
The rest of these argument names all begin with `.`  to reduce likelihood of collision with `... ` but I could use some other prefix if needed by project style conventions